commands/summary_test.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func TestSummary(t *testing.T) {
-	agentlocal.SetTransport(context.TODO(), true)
+	agentlocal.SetTransport(context.TODO(), true, 7777)
```

Comment:
Can we use a constant instead of magic number?

Reply:
Const added.